kubetest/main.go

```diff
@@ -135,6 +137,8 @@ func defineFlags() *options {
 	flag.StringVar(&o.dump, "dump", "", "If set, dump bring-up and cluster logs to this location on test or cluster-up failure")
 	flag.StringVar(&o.dumpPreTestLogs, "dump-pre-test-logs", "", "If set, dump cluster logs to this location before running tests")
 	flag.Var(&o.extract, "extract", "Extract k8s binaries from the specified release location")
+	flag.StringVar(&o.extractCIBucket, "extract-ci-bucket", "kubernetes-release-dev", "Extract k8s CI binaries from the specified GCS bucket")
+	flag.StringVar(&o.extractReleaseBucket, "extract-release-bucket", "kubernetes-release", "Extract k8s CI binaries from the specified GCS bucket")
```

Comment:
```suggestion
	flag.StringVar(&o.extractReleaseBucket, "extract-release-bucket", "kubernetes-release", "Extract k8s release binaries from the specified GCS bucket")
```

Reply:
This looks more like we need a way to override the two default hardcoded values, and not as much we need a way to customize both values in a specific run? Do we want to consider using experimental envs instead?

A more involved change would modify the corresponding strategies, but given the current maintenance state this minimal change LGTM.

Reply:
>This looks more like we need a way to override the two default hardcoded values, and not as much we need a way to customize both values in a specific run? 

Migrating kubernetes-release is going to be more involved, so I was anticipating upgrade jobs for an existing release to the in-development release pulling from kubernetes-release and k8s-release-dev in the interim.

>Do we want to consider using experimental envs instead?

I'd rather avoid envs and would prefer flags for less magic.  Though admittedly kubernetes/kubernetes scripts by themselves will need envs eventually.

> A more involved change would modify the corresponding strategies, but given the current maintenance state this minimal change LGTM.

Agreed.  The current strategies parse themselves from the string passed to extract.  I considered something like `k8s-release-dev:ci/latest`, but as you said, that seemed like it more be more invasive.